pkg/jobs/registry.go

```diff
@@ -1066,6 +1071,9 @@ func (r *Registry) Start(ctx context.Context, stopper *stop.Stopper) error {
 	})
 
 	if err := stopper.RunAsyncTask(ctx, "jobs/cancel", func(ctx context.Context) {
+		r.startedControllerTasksWG.Add(1)
```

Comment:
We have examples of both patterns in the database, but I almost always put the Add() before launching the goroutine/async task.  You don't have a guarantee that the scheduler has started the goroutine before you end up calling Wait().